client/web/web.go

```diff
@@ -426,18 +423,31 @@ func (s *Server) serveAPIAuth(w http.ResponseWriter, r *http.Request) {
 		default:
 			// no additional auth for this distro
 		}
-	case err != nil && (errors.Is(err, errNotOwner) ||
-		errors.Is(err, errNotUsingTailscale) ||
```

Comment:
Removed redundant handling of `errNotUsingTailscale` as part of this as it is handled in the above case.